docbook/reference/en/en-US/modules/StringConverter.xml

```diff
@@ -665,5 +665,152 @@ public static class TestResource {
    }
    </programlisting>
    </sect1>
+ 
+    <sect1>
+   <title>Default multiple valued <classname>ParamConverter</classname></title>
    
+   <para>
+   RESTEasy includes two built-in <classname>ParamConverter</classname>s in the resteasy-jaxrs module,
+   one for <classname>Collection</classname>s:
+   </para>
+   
+<programlisting>
+   <classname>org.jboss.resteasy.plugins.providers.MultiValuedArrayParamConverter</classname>,

```

Comment:
You mean `org.jboss.resteasy.plugins.providers.MultiValuedCollectionParamConverter` right ?

Reply:
Yep. Thanks. :-P